controllers/users.js

```diff
@@ -9,7 +9,12 @@ const UsersController = {
     const user = new User(req.body);
     user.save((err) => {
       if (err) {
-        throw err;
+        if (err.name === "MongoError" && err.code === 11000) {
+          // Duplicate email
+          return res
+            .status(422)
+            .render("error", { message: "This email already exists" });
+        }
```

Comment:
May need to review this - see comment on this PR for:
_cypress/integration/user_cant_sign_up_with_the_same_email_spec.js_